uk_bin_collection/tests/features/validate_council_outputs.feature

```diff
@@ -71,6 +71,11 @@ Feature: Test each council output matches expected results
 		| council | selenium_url | selenium_mode |
 		| BoltonCouncil | http://selenium:4444  | local  |
 
+        @BradfordMDC
+		Examples: BradfordMDC
+		| council | selenium_url | selenium_mode |
+		| BradfordMDC | http://selenium:4444  | local  |
```

Comment:
You can set the URL and mode to `None` if the council doesn't use Selenium

Reply:
I've updated the file as it's not Selenium, thanks!